src/Dfe.PlanTech.Web/Controllers/PagesController.cs

```diff
@@ -44,14 +47,30 @@ public IActionResult Error()
     => View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
 
     [HttpGet(UrlConstants.ServiceUnavailable, Name = UrlConstants.ServiceUnavailable)]
-    public IActionResult ServiceUnavailable([FromServices] IConfiguration configuration)
-     => View(new ServiceUnavailableViewModel
-     {
-         RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier,
-         ContactUsEmail = configuration["ContactUs:Email"]
-     });
+    public async Task<IActionResult> ServiceUnavailable([FromServices] IConfiguration configuration)
+    {
+        var contactLink = await _getEntityFromContentfulQuery.GetEntityById<NavigationLink>(configuration["ContactUs:LinkId"]);
```

Comment:
Needs to be added to the `appSettings` files for Dev/Test & relevant one for Staging/Prod

Reply:
[ad4a85e](https://github.com/DFE-Digital/sts-plan-technology-for-your-school/pull/873/commits/ad4a85e3c1e93001d67273e274ebdb4f18d80705) (component has the same id in dev/test and staging/prod)